va/va.h

```diff
@@ -4215,9 +4223,9 @@ typedef struct _VASurfaceDecodeMBErrors {
 } VASurfaceDecodeMBErrors;
 
 /**
- * After the application gets VA_STATUS_ERROR_DECODING_ERROR after calling vaSyncSurface(),
- * it can call vaQuerySurfaceError to find out further details on the particular error.
- * VA_STATUS_ERROR_DECODING_ERROR should be passed in as "error_status",
+ * After the application gets the VAStatus after calling vaSyncSurface(), and an error may occur,
+ * then need to call vaQuerySurfaceError() to find out further details on the particular VAStatus.
+ * The VAStatus obtained from vaSyncSurface() should be passed in as "error_status",
```

Comment:
Don't change line4218-4220; Just append some new comments as bellow:

After the applications get VA_STATUS_HW_BUSY or VA_STATUS_SUCCESSFULL from vaSyncSurface, it still can call vaQuerySurfaceError to find out further details to know if has real hw reset happened on this surface since umd and kmd could recover the context from reset with success in sometimes. VA_STATUS_ERROR_RESET should be passed in as 'error_status'.

Reply:
Done